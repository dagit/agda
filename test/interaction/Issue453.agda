
module Issue453 where

postulate
  A     : Set
  fails : {x : _} â†’ A
