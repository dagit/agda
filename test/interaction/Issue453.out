Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue453 (Issue453.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue453.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "Sort _0  [ at Issue453.agda:6,16-17 ]\n_1 : Set _0  [ at Issue453.agda:6,16-17 ]\n" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> 